Embed validation site application (#21)
datajoint/datajoint-elements: apply.md

# Apply

> Thank you for your interest in becoming a validation site for DataJoint Elements!

Reviewer: To make more general, 
```suggestion
Thank you for your interest in engineering support to for adopting DataJoint Elements!
```

> To apply to participate as a validation site, please complete the form below and submit to us.

Reviewer: ```suggestion
Please complete the form below.
```

<form name="contact-form" action="https://datajoint.io/api/element-validation-site-request" method="post">
    <section>
        <div style="float:left;margin-right:20px;">
            <label for="name">Contact Name:</label>
            <br>
            <input name="name" type="text" id="name" size="30">
        </div>
        <div style="float:left;">
            <label for="email">Contact Email:</label>
            <br>
            <input name="email" type="text" id="email" size="30">
        </div>
        <br style="clear:both;">
    </section>
    <section>
        <div style="float:left;margin-right:20px;">
            <label for="institution">Institution:</label>
            <br>
            <input name="institution" type="text" id="institution" size="30">
        </div>
        <div style="float:left;">
            <label for="lab">Lab:</label>
            <br>
            <input name="lab" type="text" id="lab" size="30">
        </div>
        <br style="clear:both;">
    </section>
    <section>
        <div style="float:left;margin-right:20px;">

>             <label for="data_modalities">Describe Data Modalities (ephys, calcium imaging, etc.):</label>

Reviewer: ```suggestion
            <label for="data_modalities">Describe data modalities (ephys, calcium imaging, etc.):</label>
```

            <br>
            <input name="data_modalities" type="text" id="data_modalities" size="67">
        </div>
        <br style="clear:both;">
    </section>
    <section>
        <div style="float:left;margin-right:20px;">
            <label for="current_workflow">Describe current workflow for data acquisition and analysis:</label>
            <br>
            <input name="current_workflow" type="text" id="current_workflow" size="67">
        </div>
        <br style="clear:both;">
    </section>
    <section>
        <div style="float:left;margin-right:20px;">

>             <label for="mysql_setup">Do you prefer to setup a MySQL Server yourself or would you like to utilize cloud managed hosting?</label>

Reviewer: ```suggestion
            <label for="mysql_setup">Do you prefer to setup a database server yourself (on premises or cloud) or use DataJoint Neuro cloud managed hosting?</label>
```

            <br>
            <input name="mysql_setup" type="text" id="mysql_setup" size="67">
        </div>
        <br style="clear:both;">
    </section>
    <section>
        <div style="float:left;margin-right:20px;">

>             <label for="start_date">When would you like to begin the tests?</label>

Reviewer: ```suggestion
            <label for="start_date">When would you like to schedule the first test?</label>
```
